src/Resources/ProductCategories.php

```diff
@@ -17,19 +19,66 @@ public function __construct(Connection $connection)
         $this->connection = $connection;
     }
 
+    // this is only returning information for the root object for now
     public function get()
     {
-        return $this->connection->get('/ProductCategories');
+        $response = $this->connection->get('/ProductCategories');
+        $body = $response->getBody();
+        return $this->getResponseFromBody($body);
     }
 
     public function post(ProductCategory $productCategory)
     {
         $payload = $productCategory->toPost();
+        $parentId = $productCategory->getParentId();
+        $urlExtension = $parentId == -1 ? '' : '/' . $parentId;
```

Comment:
Is this for the sake of ensuring we have a root category?  If so, can you provide a comment explaining the logic behind this, since we may not explicitly remember this later (and other people may use this library)?